easybuild/easyconfigs/g/GATE/GATE-10.0.2-foss-2023b.eb

```diff
@@ -0,0 +1,138 @@
+        'checksums': ['a5c32ffd397edbb829f959e7ce941ce95e9d8168b539853f1541ff02ddd603eb'],
+    }),
+    ('python-box', '6.1.0', {
+        'modulename': 'box',
+        'checksums': ['6e7c243b356cb36e2c0f0e5ed7850969fede6aa812a7f501de7768996c7744d7'],
+    }),
+    ('numpy_stl', '3.2.0', {
+        'modulename': 'stl',
+        'checksums': ['5a20c3f79cddaa0abc6a4b99f5486aceed4f88152f29b19a57acc844e183fd4d'],
+    }),
+    ('jsonpickle', '4.1.1', {
+        'checksums': ['f86e18f13e2b96c1c1eede0b7b90095bbb61d99fedc14813c44dc2f361dbbae1'],
+    }),
+    ('icrp107_database', '0.0.2', {
+        'checksums': ['82c98e6d612141bba73083187a23cf8bd8b69d875dab0718130d683e14c82217'],
+    }),
+    ('colorlog', '6.9.0', {
+        'checksums': ['bfba54a1b93b94f54e1f4fe48395725a3d92fd2a4af702f6bd70946bdc0c6ac2'],
+    }),
+    ('anytree', '2.12.1', {
+        'checksums': ['244def434ccf31b668ed282954e5d315b4e066c4940b94aff4a7962d85947830'],
+    }),
+    ('colored', '2.3.0', {
+        'checksums': ['4daf9b3ed687a01d59005e1bd03f354475226b933abb9a01836d6dde926d4c6f'],
+    }),
+    ('gatetools', '0.12.7', {
+        # skip pip check for failing local installations
+        'preinstallopts': "sed -i -e '/itk>=/d' -e '/lz4/d' -e '/xxhash/d' pyproject.toml && ",
+        'checksums': ['15e275871dc52d5d3d7410120cdf589def5f5daebe26e5fe94b5369d45c04677'],
+    }),
+    ('opengate_core', version, {
+        'preinstallopts': _core_preinstallopts,
+        'source_urls': ['https://github.com/OpenGATE/opengate/archive/'],
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': 'opengate-%(version)s.tar.gz'}],
+        'start_dir': 'core',
+        # skip 'import opengate_core' in sanity_checks - missing LD_PRELOAD - added to sanity_check_commands
+        'modulename': False,
+        'checksums': ['91551b9275c1793b61ec0b68b9ca9b4aa5e81fc16731d6ec5b4bf8f19d52c010'],
+    }),
+    ('opengate', version, {
+        # skip pip check for failing local installations
+        'preinstallopts': "sed -i '/itk/d' setup.py && ",
+        'source_urls': ['https://github.com/OpenGATE/opengate/archive/'],
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        # skip 'import opengate' in sanity_checks - missing LD_PRELOAD - added to sanity_check_commands
+        'modulename': False,
+        'checksums': ['91551b9275c1793b61ec0b68b9ca9b4aa5e81fc16731d6ec5b4bf8f19d52c010'],
+    }),
+]
+
+# link geant4-data to opengate_core dir
+postinstallcmds = [
+    "mkdir %(installdir)s/lib/python%(pyshortver)s/site-packages/opengate_core/geant4_data",
+    "ln -s $EBROOTGEANT4MINDATA/* %(installdir)s/lib/python%(pyshortver)s/site-packages/opengate_core/geant4_data",
+    "rm %(installdir)s/lib/python%(pyshortver)s/site-packages/opengate_core/geant4_data/easybuild",
+]
+
+sanity_check_commands = [
+    "export LD_PRELOAD=$EBROOTGEANT4/lib/libG4processes.so:$EBROOTGEANT4/lib/libG4geometry.so:${LD_PRELOAD} && "
```

Comment:
This should not be needed anymore, since it's baked into the module via `modextravars`?